configuration/raspi-config.md

```diff
@@ -135,51 +160,59 @@ Enable/disable shell and kernel messages on the serial connection.
 
 Enable/disable the Dallas 1-wire interface. This is usually used for DS18B20 temperature sensors.
 
+#### Remote GPIO
+
+Enable or disable remote access to the GPIO pins.
+
+### Performance Options
+
 <a name="overclock"></a> 
 ### Overclock
 
 It is possible to overclock your Raspberry Pi's CPU. The default is 700MHz but it can be set up to 1000MHz. The overclocking you can achieve will vary; overclocking too high may result in instability. Selecting this option shows the following warning:
 
 **Be aware that overclocking may reduce the lifetime of your Raspberry Pi.** If overclocking at a certain level causes system instability, try a more modest overclock. Hold down the Shift key during boot to temporarily disable overclocking.
 
-See http://elinux.org/RPi_Overclocking for more information.
+<a name="memory-split"></a>
+#### GPU Memory
 
-<a name="advanced-options"></a>
-### Advanced Options
+Change the amount of memory made available to the GPU.
 
-<a name="expand-filesystem"></a>
-#### Expand Filesystem
+#### Overlay File System
 
-If you have installed Raspberry Pi OS using NOOBS, the filesystem will have been expanded automatically. There may be a rare occasion where this is not the case, e.g. if you have copied a smaller SD card onto a larger one. In this case, you should use this option to expand your installation to fill the whole SD card, giving you more space to use for files. You will need to reboot the Raspberry Pi to make this available. Note that there is no confirmation: selecting the option begins the partition expansion immediately.
+Enable or disable a read-only filesystem
```

Comment:
This also allows you to set the boot partition read-only or not?

Reply:
No idea. I just continued on with what the menu says.

Reply:
Can probably be ignored then - this is probably one of those options where anyone using it, understands what it does :)